runtime/i18n/host-locale-provider.ts

```diff
@@ -1,3 +1,48 @@
-export {
-  SkyAppHostLocaleProvider
-} from '@skyux/i18n/modules/i18n/host-locale-provider';
+import {
+  Injectable
+} from '@angular/core';
+
+import {
+  SkyAppWindowRef
+} from '../window-ref';
+
+import {
+  Observable
+} from 'rxjs/Observable';
+
+import {
+  SkyAppLocaleInfo
+} from './locale-info';
+
+import {
+  SkyAppLocaleProvider
+} from './locale-provider';
+
+@Injectable()
+export class SkyAppHostLocaleProvider extends SkyAppLocaleProvider {
+
+  public static readonly defaultLocale = 'en-US';
+
+  constructor(
+    private windowRef: SkyAppWindowRef
```

Comment:
FYI: I had to remove `SkyAppLocaleProvider` from the constructor because Angular was barking about a circular injection. See: https://github.com/blackbaud/skyux-i18n/blob/master/src/app/public/modules/i18n/host-locale-provider.ts#L29